fix(changes-in-v14): Use Diagnostics Channel (#1426)
discordjs/guide: guide/additional-info/changes-in-v14.md

-   `invalidRequestWarning`
-   `rateLimit`

discord.js internally uses [Undici](https://github.com/nodejs/undici). Thus, the way you can monitor API requests has changed. You must now use a [Diagnostics Channel](https://undici.nodejs.org/#/docs/api/DiagnosticsChannel). Here is a simple example:

```js

> import diagnosticsChannel from 'diagnostics_channel';

Reviewer: ```suggestion
import diagnosticsChannel from 'node:diagnostics_channel';
```

diagnosticsChannel.channel('undici:request:create').subscribe(({ request }) => {
	console.log(request.method); // Log the method
	console.log(request.path); // Log the path

> 	console.log(headers); // Log the headers

Reviewer: ```suggestion
	console.log(request.headers); // Log the headers
```

-   `invalidRequestWarning`
-   `rateLimit`

> discord.js internally uses [Undici](https://github.com/nodejs/undici). Thus, the way you can monitor API requests has changed. You must now use a [Diagnostics Channel](https://undici.nodejs.org/#/docs/api/DiagnosticsChannel). Here is a simple example:

Reviewer 2: ```suggestion
discord.js now uses [Undici](https://github.com/nodejs/undici) as the underlying request handler. Thus, the way you can monitor API requests has changed. You must now use a [Diagnostics Channel](https://undici.nodejs.org/#/docs/api/DiagnosticsChannel). Here is a simple example:
```

> discord.js now uses [Undici](https://github.com/nodejs/undici) as the underlying request handler. Thus, the way you can monitor API requests has changed. You must now use a [Diagnostics Channel](https://undici.nodejs.org/#/docs/api/DiagnosticsChannel). Here is a simple example:

Reviewer 3: ```suggestion
Discord.js now uses [Undici](https://github.com/nodejs/undici) as the underlying request handler. Thus, the way you can monitor API requests has changed. You must now use a [Diagnostics Channel](https://undici.nodejs.org/#/docs/api/DiagnosticsChannel). Here is a simple example:
```

still capital at the beginning of a sentence

```js
import diagnosticsChannel from 'node:diagnostics_channel';

diagnosticsChannel.channel('undici:request:create').subscribe(data => {
	// If you use TypeScript, `data` may be casted as
	// `DiagnosticsChannel.RequestCreateMessage`
	// from Undici to receive type definitions.
	const { request } = data;
	console.log(request.method); // Log the method
	console.log(request.path); // Log the path
	console.log(request.headers); // Log the headers
	console.log(request); // Or just log everything!
});
```

> The full list of examples can be seen with the above Diagnostic Channels link.

Reviewer 2: ```suggestion
You can find further examples at the Undici Diagnostics Channel documentation link above.
```
